api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1600,8 +1600,8 @@ public void addSetMember(Concept setMember, int index) {
 		//sets have moved to the bottom and hence need to be reassigned
 		//higher sort weights than the non retired ones
 		double weight = 990.0;
-		for (ConceptSet set : sortedConceptSets) {
-			set.setSortWeight(weight += 10.0);
+		for (ConceptSet conceptSet : sortedConceptSets) {
```

Comment:
Do we have a class field named conceptSet?

Reply:
No we don't have it as class field, only as local

Reply:
Do we have a class field named set?

Reply:
We do have one. It's a Boolean

Reply:
Can you point me to it with a github link?

Reply:
https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Concept.java#LN98
line 98